  INFO root [2019-09-26 10:13:30,400] - 
  INFO root [2019-09-26 10:13:30,401] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:13:31,277] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:13:31,277] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:13:31,278] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:13:31,559] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52757 )
  INFO root [2019-09-26 10:13:31,560] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52757, SOMAXCONN=5.
  INFO root [2019-09-26 10:13:31,560] - server-configs:
  INFO root [2019-09-26 10:13:31,560] - 	internalTcpAddr : 10.15.76.77:52757
  INFO root [2019-09-26 10:13:31,560] - 	componentID : 6000
  WARN root [2019-09-26 10:13:31,561] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:13:33,301] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 15428 ----
  INFO root [2019-09-26 10:13:33,302] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:13:33,380] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:13:33,403] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:13:34,753] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:13:34,754] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:13:42,054] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:13:42,125] - Components::findComponents: found logger, addr:10.15.76.77:52759
  INFO root [2019-09-26 10:13:42,125] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:13:42,125] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:42,220] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:13:43,285] - Forwarding logs to logger[10.15.76.77:52759/0/0/0]...
 ERROR root [2019-09-26 10:20:13,022] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144976692, lastReceivedTime=3304039984629, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 10:20:13,035] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:20:13,035] - ==> [2019-09-26 10:20:13 018] TCPPacketReceiver::processPendingEvents(10.15.76.77:52759): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:20:13,036] - ==> [2019-09-26 10:20:13 023] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144976692, lastReceivedTime=3304039984629, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-26 10:20:13,120] - TCPPacketReceiver::processPendingEvents(10.15.76.77:52760): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:20:13,133] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304146084575, lastReceivedTime=3304058375340, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 10:20:13,158] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 10:20:13,158] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:20:13,158] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 10:20:13,159] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 10:37:35,472] - 
  INFO root [2019-09-26 10:37:35,479] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:37:35,481] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:37:35,486] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:37:35,487] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:37:35,522] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52922 )
  INFO root [2019-09-26 10:37:35,525] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52922, SOMAXCONN=5.
  INFO root [2019-09-26 10:37:35,525] - server-configs:
  INFO root [2019-09-26 10:37:35,525] - 	internalTcpAddr : 10.15.76.77:52922
  INFO root [2019-09-26 10:37:35,526] - 	componentID : 6000
  WARN root [2019-09-26 10:37:35,534] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:37:37,054] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 13888 ----
  INFO root [2019-09-26 10:37:37,055] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:37:37,055] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:37:37,056] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:37:37,465] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:37:37,465] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:37:44,657] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:37:44,672] - Components::findComponents: found logger, addr:10.15.76.77:52920
  INFO root [2019-09-26 10:37:44,672] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:37:44,672] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:37:44,673] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:37:44,777] - Components::findComponents: found dbmgr, addr:10.15.76.77:52918
  INFO root [2019-09-26 10:37:44,777] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:37:44,777] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:37:44,784] - Components::findComponents: found dbmgr, addr:10.15.76.77:52918
  WARN root [2019-09-26 10:37:45,784] - Forwarding logs to logger[10.15.76.77:52920/0/0/0]...
 ERROR root [2019-09-26 10:40:29,454] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309295912, lastReceivedTime=3316069021726, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-26 10:40:29,457] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:40:29,457] - ==> [2019-09-26 10:40:29 440] TCPPacketReceiver::processPendingEvents(10.15.76.77:52920): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:40:29,458] - ==> [2019-09-26 10:40:29 455] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309295912, lastReceivedTime=3316069021726, inactivityExceptionPeriod=599499489)
  WARN root [2019-09-26 10:40:29,722] - TCPPacketReceiver::processPendingEvents(10.15.76.77:52918): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:40:29,730] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316312053484, lastReceivedTime=3316067138949, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-26 10:40:29,738] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 10:40:29,739] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:40:29,739] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 10:40:29,740] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:40:30,041] - ThreadPool::destroy(): waiting for thread(2)[0x1494b08,0x14944f0,], try=1
  INFO root [2019-09-26 10:44:55,780] - 
  INFO root [2019-09-26 10:44:55,781] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:44:55,782] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:44:55,782] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:44:55,782] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:44:55,792] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53013 )
  INFO root [2019-09-26 10:44:55,793] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53013, SOMAXCONN=5.
  INFO root [2019-09-26 10:44:55,793] - server-configs:
  INFO root [2019-09-26 10:44:55,793] - 	internalTcpAddr : 10.15.76.77:53013
  INFO root [2019-09-26 10:44:55,793] - 	componentID : 6000
  WARN root [2019-09-26 10:44:55,794] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:44:57,300] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 11284 ----
  INFO root [2019-09-26 10:44:57,300] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:44:57,301] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:44:57,301] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:44:57,712] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:44:57,714] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:45:04,903] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:45:04,952] - Components::findComponents: found logger, addr:10.15.76.77:53012
  INFO root [2019-09-26 10:45:04,952] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:04,952] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:45:04,953] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:45:05,057] - Components::findComponents: found dbmgr, addr:10.15.76.77:53016
  INFO root [2019-09-26 10:45:05,057] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:05,057] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:45:05,057] - Components::findComponents: found dbmgr, addr:10.15.76.77:53016
  WARN root [2019-09-26 10:45:06,057] - Forwarding logs to logger[10.15.76.77:53012/0/0/0]...
 ERROR root [2019-09-26 10:47:13,905] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353810782, lastReceivedTime=3320172195357, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 10:47:13,929] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:47:13,929] - ==> [2019-09-26 10:47:13 890] TCPPacketReceiver::processPendingEvents(10.15.76.77:53012): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:47:13,930] - ==> [2019-09-26 10:47:13 907] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353810782, lastReceivedTime=3320172195357, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-26 10:47:14,100] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53016): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:47:14,140] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320356151439, lastReceivedTime=3320175396769, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 10:47:14,154] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 10:47:14,159] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:47:14,160] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 10:47:14,160] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:47:14,463] - ThreadPool::destroy(): waiting for thread(2)[0x1358d50,0x1358668,], try=1
  INFO root [2019-09-26 10:53:20,517] - 
  INFO root [2019-09-26 10:53:20,518] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:20,520] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:20,520] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:20,520] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:20,553] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53113 )
  INFO root [2019-09-26 10:53:20,553] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53113, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:20,553] - server-configs:
  INFO root [2019-09-26 10:53:20,553] - 	internalTcpAddr : 10.15.76.77:53113
  INFO root [2019-09-26 10:53:20,553] - 	componentID : 6000
  WARN root [2019-09-26 10:53:20,557] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:22,062] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 2408 ----
  INFO root [2019-09-26 10:53:22,063] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:22,064] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:22,064] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:22,488] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:53:22,493] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:53:29,657] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:53:29,678] - Components::findComponents: found logger, addr:10.15.76.77:53111
  INFO root [2019-09-26 10:53:29,679] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:53:29,679] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:53:29,684] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:53:30,784] - Forwarding logs to logger[10.15.76.77:53111/0/0/0]...
 ERROR root [2019-09-26 10:53:37,356] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324188317948, lastReceivedTime=3324111591427, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:53:37,358] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:53:37,358] - ==> [2019-09-26 10:53:37 354] TCPPacketReceiver::processPendingEvents(10.15.76.77:53111): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:53:37,358] - ==> [2019-09-26 10:53:37 357] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324188317948, lastReceivedTime=3324111591427, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:53:39,033] - 
  INFO root [2019-09-26 10:53:39,052] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:39,053] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:39,053] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:39,053] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:39,100] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53132 )
  INFO root [2019-09-26 10:53:39,100] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53132, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:39,100] - server-configs:
  INFO root [2019-09-26 10:53:39,100] - 	internalTcpAddr : 10.15.76.77:53132
  INFO root [2019-09-26 10:53:39,100] - 	componentID : 6000
  WARN root [2019-09-26 10:53:39,101] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:40,623] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 15892 ----
  INFO root [2019-09-26 10:53:40,624] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:40,624] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:40,625] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:41,066] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:53:41,066] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:53:48,267] - Components::findComponents: find logger(1)...
  WARN root [2019-09-26 10:53:48,607] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20989, errorno=10048
  INFO root [2019-09-26 10:53:49,626] - Components::findComponents: find logger(2)...
  INFO root [2019-09-26 10:53:49,685] - Components::findComponents: found logger, addr:10.15.76.77:53128
  INFO root [2019-09-26 10:53:49,685] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:53:49,685] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:53:49,687] - Components::findComponents: find dbmgr(3)...
  WARN root [2019-09-26 10:53:50,789] - Forwarding logs to logger[10.15.76.77:53128/0/0/0]...
 ERROR root [2019-09-26 10:53:55,025] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324365008858, lastReceivedTime=3324311623680, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 10:53:55,026] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:53:55,026] - ==> [2019-09-26 10:53:55 025] TCPPacketReceiver::processPendingEvents(10.15.76.77:53128): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:53:55,027] - ==> [2019-09-26 10:53:55 026] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324365008858, lastReceivedTime=3324311623680, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 10:53:56,950] - 
  INFO root [2019-09-26 10:53:56,954] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:56,956] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:56,956] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:56,956] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:57,036] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53148 )
  INFO root [2019-09-26 10:53:57,037] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53148, SOMAXCONN=5.
  INFO root [2019-09-26 10:53:57,037] - server-configs:
  INFO root [2019-09-26 10:53:57,037] - 	internalTcpAddr : 10.15.76.77:53148
  INFO root [2019-09-26 10:53:57,037] - 	componentID : 6000
  WARN root [2019-09-26 10:53:57,042] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:58,550] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 16268 ----
  INFO root [2019-09-26 10:53:58,551] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:58,551] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:58,551] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:58,960] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:53:58,960] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 10:53:59,310] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20120, errorno=10048
  INFO root [2019-09-26 10:54:06,493] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:54:06,592] - Components::findComponents: found logger, addr:10.15.76.77:53146
  INFO root [2019-09-26 10:54:06,592] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:54:06,593] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:54:06,594] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:54:06,595] - Components::findComponents: found logger, addr:10.15.76.77:53146
  WARN root [2019-09-26 10:54:07,700] - Forwarding logs to logger[10.15.76.77:53146/0/0/0]...
 ERROR root [2019-09-26 10:54:10,037] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324515132744, lastReceivedTime=3324480698721, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 10:54:10,040] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:54:10,041] - ==> [2019-09-26 10:54:10 029] TCPPacketReceiver::processPendingEvents(10.15.76.77:53146): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:54:10,041] - ==> [2019-09-26 10:54:10 038] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324515132744, lastReceivedTime=3324480698721, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 10:54:11,816] - 
  INFO root [2019-09-26 10:54:11,817] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:54:11,818] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:54:11,818] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:54:11,818] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:54:11,828] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53167 )
  INFO root [2019-09-26 10:54:11,829] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53167, SOMAXCONN=5.
  INFO root [2019-09-26 10:54:11,829] - server-configs:
  INFO root [2019-09-26 10:54:11,829] - 	internalTcpAddr : 10.15.76.77:53167
  INFO root [2019-09-26 10:54:11,829] - 	componentID : 6000
  WARN root [2019-09-26 10:54:11,830] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:54:13,335] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 3140 ----
  INFO root [2019-09-26 10:54:13,336] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:54:13,336] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:54:13,336] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:54:13,782] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:54:13,801] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 10:54:14,224] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20058, errorno=10048
  INFO root [2019-09-26 10:54:21,399] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:54:21,403] - Components::findComponents: found logger, addr:10.15.76.77:53169
  INFO root [2019-09-26 10:54:21,404] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:54:21,404] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:54:21,405] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:54:22,507] - Forwarding logs to logger[10.15.76.77:53169/0/0/0]...
 ERROR root [2019-09-26 10:54:28,762] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324702374538, lastReceivedTime=3324628808152, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-26 10:54:28,763] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:54:28,763] - ==> [2019-09-26 10:54:28 761] TCPPacketReceiver::processPendingEvents(10.15.76.77:53169): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:54:28,763] - ==> [2019-09-26 10:54:28 762] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324702374538, lastReceivedTime=3324628808152, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-26 10:56:35,175] - 
  INFO root [2019-09-26 10:56:35,181] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:56:35,183] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:56:35,183] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:56:35,183] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:56:35,245] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53186 )
  INFO root [2019-09-26 10:56:35,245] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53186, SOMAXCONN=5.
  INFO root [2019-09-26 10:56:35,245] - server-configs:
  INFO root [2019-09-26 10:56:35,245] - 	internalTcpAddr : 10.15.76.77:53186
  INFO root [2019-09-26 10:56:35,245] - 	componentID : 6000
  WARN root [2019-09-26 10:56:35,254] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:56:36,762] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 9660 ----
  INFO root [2019-09-26 10:56:36,763] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:56:36,764] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:56:36,764] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:56:37,165] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 10:56:37,187] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:56:44,387] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:56:44,439] - Components::findComponents: found logger, addr:10.15.76.77:53185
  INFO root [2019-09-26 10:56:44,439] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:44,439] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:56:44,441] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:56:44,543] - Components::findComponents: found dbmgr, addr:10.15.76.77:53182
  INFO root [2019-09-26 10:56:44,543] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:44,543] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:56:44,550] - Components::findComponents: found dbmgr, addr:10.15.76.77:53182
  WARN root [2019-09-26 10:56:45,551] - Forwarding logs to logger[10.15.76.77:53185/0/0/0]...
 ERROR root [2019-09-26 10:58:37,838] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193140361, lastReceivedTime=3327166852021, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:58:37,843] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:58:37,844] - ==> [2019-09-26 10:58:37 831] TCPPacketReceiver::processPendingEvents(10.15.76.77:53185): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:58:37,844] - ==> [2019-09-26 10:58:37 839] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193140361, lastReceivedTime=3327166852021, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-26 10:58:37,948] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53182): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:58:37,951] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327194271013, lastReceivedTime=3327165921975, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:58:37,981] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 10:58:37,982] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:58:37,982] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 10:58:37,983] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:00:58,148] - 
  INFO root [2019-09-26 11:00:58,150] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:00:58,152] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:00:58,152] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:00:58,152] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:00:58,191] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53258 )
  INFO root [2019-09-26 11:00:58,192] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53258, SOMAXCONN=5.
  INFO root [2019-09-26 11:00:58,192] - server-configs:
  INFO root [2019-09-26 11:00:58,192] - 	internalTcpAddr : 10.15.76.77:53258
  INFO root [2019-09-26 11:00:58,192] - 	componentID : 6000
  WARN root [2019-09-26 11:00:58,214] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:00:59,719] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 19780 ----
  INFO root [2019-09-26 11:00:59,720] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:00:59,720] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:00:59,721] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:01:00,226] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:01:00,233] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:01:07,462] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:01:07,479] - Components::findComponents: found logger, addr:10.15.76.77:53251
  INFO root [2019-09-26 11:01:07,479] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:07,479] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:01:07,480] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:01:07,583] - Components::findComponents: found dbmgr, addr:10.15.76.77:53254
  INFO root [2019-09-26 11:01:07,583] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:07,583] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:01:07,583] - Components::findComponents: found dbmgr, addr:10.15.76.77:53254
  WARN root [2019-09-26 11:01:08,584] - Forwarding logs to logger[10.15.76.77:53251/0/0/0]...
 ERROR root [2019-09-26 11:02:47,359] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688345644, lastReceivedTime=3329495437248, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 11:02:47,366] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:02:47,366] - ==> [2019-09-26 11:02:47 354] TCPPacketReceiver::processPendingEvents(10.15.76.77:53251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:02:47,367] - ==> [2019-09-26 11:02:47 360] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688345644, lastReceivedTime=3329495437248, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-26 11:02:47,470] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53254): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:02:47,476] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329689519495, lastReceivedTime=3329496192960, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 11:02:47,499] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:02:47,505] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:02:47,506] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:02:47,506] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:02:49,423] - 
  INFO root [2019-09-26 11:02:49,424] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:02:49,425] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:02:49,426] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:02:49,426] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:02:49,440] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53304 )
  INFO root [2019-09-26 11:02:49,440] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53304, SOMAXCONN=5.
  INFO root [2019-09-26 11:02:49,440] - server-configs:
  INFO root [2019-09-26 11:02:49,440] - 	internalTcpAddr : 10.15.76.77:53304
  INFO root [2019-09-26 11:02:49,440] - 	componentID : 6000
  WARN root [2019-09-26 11:02:49,440] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:02:50,946] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 9760 ----
  INFO root [2019-09-26 11:02:50,948] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:02:50,948] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:02:50,949] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:02:51,460] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:02:51,461] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 11:02:57,703] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53326): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-26 11:02:58,710] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:02:58,808] - Components::findComponents: found logger, addr:10.15.76.77:53306
  INFO root [2019-09-26 11:02:58,808] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:02:58,808] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:02:58,812] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:02:58,913] - Components::findComponents: found dbmgr, addr:10.15.76.77:53309
  INFO root [2019-09-26 11:02:58,913] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:02:58,913] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:02:58,914] - Components::findComponents: found dbmgr, addr:10.15.76.77:53309
  WARN root [2019-09-26 11:02:59,914] - Forwarding logs to logger[10.15.76.77:53306/0/0/0]...
 ERROR root [2019-09-26 11:04:12,360] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538357959, lastReceivedTime=3330309277999, inactivityExceptionPeriod=599498594)
  INFO root [2019-09-26 11:04:12,362] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:04:12,362] - ==> [2019-09-26 11:04:12 356] TCPPacketReceiver::processPendingEvents(10.15.76.77:53306): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:04:12,362] - ==> [2019-09-26 11:04:12 361] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538357959, lastReceivedTime=3330309277999, inactivityExceptionPeriod=599498594)
  WARN root [2019-09-26 11:04:12,463] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53309): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:04:12,469] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330539456839, lastReceivedTime=3330311130445, inactivityExceptionPeriod=599498594)
  INFO root [2019-09-26 11:04:12,481] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:04:12,486] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:04:12,487] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:04:12,487] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:04:12,789] - ThreadPool::destroy(): waiting for thread(2)[0x17b4760,0x17b4a38,], try=1
  INFO root [2019-09-26 11:06:52,695] - 
  INFO root [2019-09-26 11:06:52,696] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:06:52,697] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:06:52,698] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:06:52,698] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:06:52,710] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53385 )
  INFO root [2019-09-26 11:06:52,710] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53385, SOMAXCONN=5.
  INFO root [2019-09-26 11:06:52,711] - server-configs:
  INFO root [2019-09-26 11:06:52,711] - 	internalTcpAddr : 10.15.76.77:53385
  INFO root [2019-09-26 11:06:52,711] - 	componentID : 6000
  WARN root [2019-09-26 11:06:52,738] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:06:54,259] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 17032 ----
  INFO root [2019-09-26 11:06:54,260] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:06:54,260] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:06:54,261] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:06:54,722] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:06:54,729] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:07:01,922] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:07:01,929] - Components::findComponents: found logger, addr:10.15.76.77:53383
  INFO root [2019-09-26 11:07:01,929] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:01,930] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:01,931] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:07:02,034] - Components::findComponents: found dbmgr, addr:10.15.76.77:53381
  INFO root [2019-09-26 11:07:02,035] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:02,035] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:02,044] - Components::findComponents: found dbmgr, addr:10.15.76.77:53381
  WARN root [2019-09-26 11:07:03,045] - Forwarding logs to logger[10.15.76.77:53383/0/0/0]...
 ERROR root [2019-09-26 11:08:49,376] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308514553, lastReceivedTime=3333041705406, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-26 11:08:49,385] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:08:49,386] - ==> [2019-09-26 11:08:49 372] TCPPacketReceiver::processPendingEvents(10.15.76.77:53383): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:08:49,386] - ==> [2019-09-26 11:08:49 376] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308514553, lastReceivedTime=3333041705406, inactivityExceptionPeriod=599499617)
  WARN root [2019-09-26 11:08:49,489] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53381): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:08:49,491] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333309674346, lastReceivedTime=3333041095838, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-26 11:08:49,493] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:08:49,494] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:08:49,494] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:08:49,494] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:08:51,379] - 
  INFO root [2019-09-26 11:08:51,380] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:08:51,381] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:08:51,381] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:08:51,381] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:08:51,392] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53455 )
  INFO root [2019-09-26 11:08:51,393] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53455, SOMAXCONN=5.
  INFO root [2019-09-26 11:08:51,393] - server-configs:
  INFO root [2019-09-26 11:08:51,393] - 	internalTcpAddr : 10.15.76.77:53455
  INFO root [2019-09-26 11:08:51,393] - 	componentID : 6000
  WARN root [2019-09-26 11:08:51,393] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:08:52,898] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 13000 ----
  INFO root [2019-09-26 11:08:52,898] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:08:52,899] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:08:52,900] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:08:53,345] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:08:53,345] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:09:00,561] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:09:00,574] - Components::findComponents: found logger, addr:10.15.76.77:53452
  INFO root [2019-09-26 11:09:00,574] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:09:00,574] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:09:00,576] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:09:00,679] - Components::findComponents: found dbmgr, addr:10.15.76.77:53454
  INFO root [2019-09-26 11:09:00,679] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:09:00,679] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:09:00,689] - Components::findComponents: found dbmgr, addr:10.15.76.77:53454
  WARN root [2019-09-26 11:09:01,689] - Forwarding logs to logger[10.15.76.77:53452/0/0/0]...
 ERROR root [2019-09-26 11:10:11,691] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131668958, lastReceivedTime=3333927989295, inactivityExceptionPeriod=599498913)
  INFO root [2019-09-26 11:10:11,696] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:10:11,697] - ==> [2019-09-26 11:10:11 690] TCPPacketReceiver::processPendingEvents(10.15.76.77:53452): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:10:11,697] - ==> [2019-09-26 11:10:11 692] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131668958, lastReceivedTime=3333927989295, inactivityExceptionPeriod=599498913)
  WARN root [2019-09-26 11:10:11,793] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53454): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:10:11,795] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334132709455, lastReceivedTime=3333928311081, inactivityExceptionPeriod=599498913)
  INFO root [2019-09-26 11:10:11,801] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:10:11,802] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:10:11,803] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:10:11,804] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:10:13,576] - 
  INFO root [2019-09-26 11:10:13,579] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:10:13,580] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:10:13,580] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:10:13,580] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:10:13,602] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53492 )
  INFO root [2019-09-26 11:10:13,603] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53492, SOMAXCONN=5.
  INFO root [2019-09-26 11:10:13,603] - server-configs:
  INFO root [2019-09-26 11:10:13,603] - 	internalTcpAddr : 10.15.76.77:53492
  INFO root [2019-09-26 11:10:13,603] - 	componentID : 6000
  WARN root [2019-09-26 11:10:13,605] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:10:15,124] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 15612 ----
  INFO root [2019-09-26 11:10:15,124] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:10:15,125] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:10:15,125] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:10:15,544] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:10:15,545] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:10:22,727] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:10:22,792] - Components::findComponents: found logger, addr:10.15.76.77:53490
  INFO root [2019-09-26 11:10:22,792] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:22,792] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:22,806] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:10:22,910] - Components::findComponents: found dbmgr, addr:10.15.76.77:53493
  INFO root [2019-09-26 11:10:22,910] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:22,910] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:22,911] - Components::findComponents: found dbmgr, addr:10.15.76.77:53493
  WARN root [2019-09-26 11:10:23,911] - Forwarding logs to logger[10.15.76.77:53490/0/0/0]...
 ERROR root [2019-09-26 11:12:29,803] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512787590, lastReceivedTime=3335349686499, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:12:29,805] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:12:29,805] - ==> [2019-09-26 11:12:29 802] TCPPacketReceiver::processPendingEvents(10.15.76.77:53490): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:12:29,805] - ==> [2019-09-26 11:12:29 804] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512787590, lastReceivedTime=3335349686499, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-26 11:12:29,902] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53493): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:12:29,903] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335513791651, lastReceivedTime=3335351222473, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:12:29,921] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:12:29,934] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:12:29,934] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:12:29,934] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:12:31,557] - 
  INFO root [2019-09-26 11:12:31,558] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:12:31,559] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:12:31,559] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:12:31,559] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:12:31,591] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53567 )
  INFO root [2019-09-26 11:12:31,591] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53567, SOMAXCONN=5.
  INFO root [2019-09-26 11:12:31,592] - server-configs:
  INFO root [2019-09-26 11:12:31,592] - 	internalTcpAddr : 10.15.76.77:53567
  INFO root [2019-09-26 11:12:31,592] - 	componentID : 6000
  WARN root [2019-09-26 11:12:31,599] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:12:33,115] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 18288 ----
  INFO root [2019-09-26 11:12:33,115] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:12:33,116] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:12:33,116] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:12:33,519] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:12:33,530] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:12:40,718] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:12:40,779] - Components::findComponents: found logger, addr:10.15.76.77:53572
  INFO root [2019-09-26 11:12:40,779] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:40,779] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:12:40,780] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:12:40,786] - Components::findComponents: found dbmgr, addr:10.15.76.77:53566
  INFO root [2019-09-26 11:12:40,786] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:40,787] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:12:40,795] - Components::findComponents: found logger, addr:10.15.76.77:53572
  INFO root [2019-09-26 11:12:40,801] - Components::findComponents: found dbmgr, addr:10.15.76.77:53566
  WARN root [2019-09-26 11:12:41,801] - Forwarding logs to logger[10.15.76.77:53572/0/0/0]...
 ERROR root [2019-09-26 11:15:24,609] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260850798, lastReceivedTime=3337030902918, inactivityExceptionPeriod=599503134)
  INFO root [2019-09-26 11:15:24,613] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:15:24,614] - ==> [2019-09-26 11:15:24 607] TCPPacketReceiver::processPendingEvents(10.15.76.77:53572): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:15:24,614] - ==> [2019-09-26 11:15:24 610] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260850798, lastReceivedTime=3337030902918, inactivityExceptionPeriod=599503134)
  WARN root [2019-09-26 11:15:24,700] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53566): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:15:24,701] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337261774038, lastReceivedTime=3337030397128, inactivityExceptionPeriod=599503134)
  INFO root [2019-09-26 11:15:24,711] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:15:24,719] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:15:24,719] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:15:24,720] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:15:26,431] - 
  INFO root [2019-09-26 11:15:26,432] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:15:26,433] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:15:26,433] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:15:26,433] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:15:26,443] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53630 )
  INFO root [2019-09-26 11:15:26,444] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53630, SOMAXCONN=5.
  INFO root [2019-09-26 11:15:26,444] - server-configs:
  INFO root [2019-09-26 11:15:26,444] - 	internalTcpAddr : 10.15.76.77:53630
  INFO root [2019-09-26 11:15:26,444] - 	componentID : 6000
  WARN root [2019-09-26 11:15:26,444] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:15:27,949] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 7136 ----
  INFO root [2019-09-26 11:15:27,950] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:15:27,950] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:15:27,951] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:15:28,360] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:15:28,361] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:15:35,532] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:15:35,612] - Components::findComponents: found logger, addr:10.15.76.77:53628
  INFO root [2019-09-26 11:15:35,613] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:15:35,613] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:15:35,614] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:15:35,619] - Components::findComponents: found dbmgr, addr:10.15.76.77:53629
  INFO root [2019-09-26 11:15:35,619] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:15:35,619] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:15:35,721] - Components::findComponents: found dbmgr, addr:10.15.76.77:53629
  WARN root [2019-09-26 11:15:36,722] - Forwarding logs to logger[10.15.76.77:53628/0/0/0]...
 ERROR root [2019-09-26 11:18:22,248] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037242953, lastReceivedTime=3338778392128, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-26 11:18:22,252] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:18:22,252] - ==> [2019-09-26 11:18:22 246] TCPPacketReceiver::processPendingEvents(10.15.76.77:53628): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:18:22,253] - ==> [2019-09-26 11:18:22 249] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037242953, lastReceivedTime=3338778392128, inactivityExceptionPeriod=599500320)
  WARN root [2019-09-26 11:18:22,360] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53629): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:18:22,362] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339038378703, lastReceivedTime=3338778956539, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-26 11:18:22,393] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:18:22,404] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:18:22,404] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:18:22,405] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-26 11:19:32,869] - 
  INFO root [2019-09-26 11:19:32,869] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:19:32,870] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:19:32,871] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:19:32,871] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:19:32,880] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53699 )
  INFO root [2019-09-26 11:19:32,881] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53699, SOMAXCONN=5.
  INFO root [2019-09-26 11:19:32,881] - server-configs:
  INFO root [2019-09-26 11:19:32,881] - 	internalTcpAddr : 10.15.76.77:53699
  INFO root [2019-09-26 11:19:32,881] - 	componentID : 6000
  WARN root [2019-09-26 11:19:32,881] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:19:34,390] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 17024 ----
  INFO root [2019-09-26 11:19:34,390] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:19:34,391] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:19:34,391] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:19:34,823] - ---- cellappmgr is running ----
 DEBUG root [2019-09-26 11:19:34,824] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:19:42,053] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:19:42,071] - Components::findComponents: found logger, addr:10.15.76.77:53696
  INFO root [2019-09-26 11:19:42,072] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:19:42,072] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:19:42,073] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:19:42,074] - Components::findComponents: found logger, addr:10.15.76.77:53696
  INFO root [2019-09-26 11:19:42,081] - Components::findComponents: found dbmgr, addr:10.15.76.77:53701
  INFO root [2019-09-26 11:19:42,082] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:19:42,082] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:19:42,186] - Components::findComponents: found dbmgr, addr:10.15.76.77:53701
  WARN root [2019-09-26 11:19:43,186] - Forwarding logs to logger[10.15.76.77:53696/0/0/0]...
 ERROR root [2019-09-26 11:21:25,809] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872848273, lastReceivedTime=3340642842588, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:21:25,812] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:21:25,812] - ==> [2019-09-26 11:21:25 807] TCPPacketReceiver::processPendingEvents(10.15.76.77:53696): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:21:25,813] - ==> [2019-09-26 11:21:25 810] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872848273, lastReceivedTime=3340642842588, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-26 11:21:25,913] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53701): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:21:25,915] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340873911821, lastReceivedTime=3340644387022, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:21:25,927] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-26 11:21:25,930] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:21:25,930] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-26 11:21:25,930] - ThreadPool::destroy(): starting size 2.
